packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -27,8 +27,12 @@ const arrowStrokeWidth = 1.5;
 /// Hardcoded sizes for scaling the flex children widget properly.
 const minRenderWidth = 215.0;
 const minRenderHeight = 275.0;
-const defaultMaxRenderWidth = 300.0;
-const defaultMaxRenderHeight = 300.0;
+
+/// The size to shrink a widget by when animating it in.
+const entranceMargin = 60;
```

Comment:
nit: 60.0 to avoid casting into double since margin is double?

Reply:
Good idea.